Project_1.cpp

```diff
@@ -41,19 +37,15 @@
             }
 
             window.clear();
-
+            
             for (int i = 0; i < GRID_SIZE; ++i)
             {
                 for (int j = 0; j < GRID_SIZE; ++j)
                 {
                     cell.setPosition(i * GRID_SIZE, j * GRID_SIZE);

```

Comment:
GRID_SIZE does not look like it is the correct variable you want to use.
I imagine that you want to fill the entire window with colors.

GRID_SIZE has a value of 5 (which would mean 5 pixels if you set the Position with this value).
CELL_SIZE has a value that is based on how big the window is divided by grid number. I think you would want to do this.

Position is set by pixels. I would highly suggest you to imagine what the code does while writing your code. Helps give you an idea what your code potentially does with the variables you use.

Reply:
Also, GRID_SIZE might not be the best variable name to indicate the usage of this. I would suggest **NUM_OF_COLUMNS** or **COLUMN_COUNT**.